package.json

```diff
@@ -35,23 +35,26 @@
     "bootstrap-sass": "^3.3.6",
     "classnames": "^2.2.3",
     "crypto-js": "^3.1.5",
+    "cyverse-ui": "^0.*.*",
     "d3": "^3.5.6",
     "highcharts-commonjs": "^4.1.7-1",
     "history": "^2.1.2",
     "jquery": "^2.1.3",
     "jstz": "^1.0.7",
+    "lodash": "^4.17.4",
```

Comment:
Where does the `lodash` dependency come in? If it is a dependency of `material-ui`, then it will be installed as a transitive dependency (and should not be included in our package.json).